### ISSUE TITLE
Enum literals in a language are not deserialized

### ISSUE THREAD

**Issue by woppa684** [LIonWeb-org/lioncore-typescript#58]
The resulting Enumeration  has an empty array for literals, although the langugae contains the proper children.

**Comment by dslmeinte** [comment]
Could you maybe provide a failing unit test?

**Comment by dslmeinte** [comment]
Oh, and how are you using the TypeScript code? Through the NPM package (and then version 0.2.0, I presume), or (most likely:) directly from the Git repo?

**Comment by woppa684** [comment]
I am using the git repo version. I will try to make a unit test for this. The error could also be in the way the MPS version provides the data of course and I'; m not sure how to set that up in a test. I'll have a look!

**Comment by dslmeinte** [comment]
For comparison, you might want to have a look at: https://github.com/LIonWeb-org/lioncore-typescript/commit/d894020b660ed495aab9a42558dae4f3ad645c02#diff-16eaae6fd313f56618771fcde983803881bc7e1c1fa879a97eda18a3a01fe689R12

**Comment by woppa684** [comment]
By the way, until now I've been copying the source code and adapting it to make it compile in our environment, since I cannot get the NPM package (after building it) to work in our codebase. It's an ESM only package but both our IDE (VSCode) and our build environment (Nx workspace / Angular CLI) don't know how to handle it apparently. For example, after adding an `index.d.ts` file in the root of the package, the IDE and the build pick it up but exported functions still don't work then. I also need a lot more stuff exported then is exported by default by this package so I adapted the mod.ts file anyway of course.

Adapting the source code to something that works for us is quite an effort (in our company, we don't use Deno for example, I don't know anybody who does ;)) so I haven't done that with the latest version yet. I will try that first because this issue might have already been fixed (the unit test looks good I would say).
